checkstyle-sonar-plugin/pom.xml

```diff
@@ -330,6 +330,11 @@
                 <branchRate>100</branchRate>
                 <lineRate>78</lineRate>
               </regex>
+              <regex>
+                <pattern>org.sonar.plugins.checkstyle.CheckstyleExecutor</pattern>
+                <branchRate>90</branchRate>
+                <lineRate>94</lineRate>
+              </regex>
```

Comment:
That would mean removing Java 7 try-with-resources. try-with-resources generates a lot of hidden branches (null checks, suppressed exceptions, …) that make it almost impossible to reach 100% coverage.

I have no problem making this change, I just wanted to point this out.